runtime/src/main/java/jp/co/soramitsu/runtime/chain/model/Chain.kt

```diff
@@ -0,0 +1,32 @@
+package jp.co.soramitsu.runtime.chain.model
+
+data class Chain(
+    val id: String,
+    val name: String,
+    val assets: List<Asset>,
+    val nodes: List<Node>,
+    val icon: String,
+    val addressPrefix: Int,
+    val types: Types?,
+    val isEthereumBased: Boolean,
+    val isTestNet: Boolean,
+    val parentId: String?,
+) {
+
+    data class Types(
+        val url: String,
+        val overridesCommon: Boolean,
+    )
+
+    data class Asset(
+        val id: Int,
```

Comment:
In iOS version we also add optional icon (URL) for the cases when asset's icon is different from the network one.